homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -109,6 +110,15 @@ def options(self) -> dict[str, Any]:
         """Return the options linked to the current flow handler."""
         return self._options
 
+    @property
+    def local_context(self) -> dict[str, Any]:
```

Comment:
What is the different with the existing `context?

Reply:
There is currently no context in `SchemaCommonFlowHandler`

There is a context in `SchemaCommonFlowHandler.parent_handler` (the config flow or the options flow) but this gets passed around to frontend and FlowResult. In this case we only want to use it for temporary data within the `SchemaCommonFlowHandler`

Use case is in #82926

Reply:
> There is currently no context in SchemaCommonFlowHandler

I understand, but question was more: why is it called `local_context` here and `context` in other places...

It must be different then, I guess?

../Frenck

Reply:
Right, the existing `context` in `FlowHandler` has absolutely nothing at all to do with the `local_context` being added to `SchemaCommonFlowHandler` in this PR, but I agree the naming suggests otherwise.

@epenet Maybe name the new property `flow_state` or something, since that's what the new property is meant to hold?

Reply:
> has absolutely nothing at all to do with the 

They serve similar purposes as it seems. But as said, the difference is not completely clear to me, nor can I make that up from the existing documentation.

Reply:
Happy to rename => done in 9297c80